trl/trainer/dpo_trainer.py

```diff
@@ -1225,7 +1241,7 @@ def get_batch_loss_metrics(
         reward_accuracies = (chosen_rewards > rejected_rewards).float()
 
         if self.args.rpo_alpha is not None:
-            losses = losses * self.args.rpo_alpha - policy_chosen_logps_avg
+            losses = losses * self.args.rpo_alpha + policy_nll_loss
```

Comment:
Perhaps I'm missing something, but isn't the NLL loss normalized by the total sequence length? 


<img width="987" alt="Screenshot 2024-06-06 at 14 54 39" src="https://github.com/huggingface/trl/assets/26859204/a5638d76-8fb8-4062-9af8-fc12606cfa28">

Reply:
yes the `CrossEntropyLoss` is reducing by `mean` by default